adam-r/pom.xml

```diff
@@ -23,6 +23,20 @@
         <artifactId>exec-maven-plugin</artifactId>
         <executions>
           <execution>
+            <id>doc-r</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+            <configuration>
+              <executable>R</executable>
+              <arguments>
+                <argument>-e</argument>
+                <argument>"library(devtools);devtools::document()"</argument>
```

Comment:
This should require a mention in https://github.com/bigdatagenomics/adam/blob/master/docs/source/02_installation.md#building-for-r-r-build of additional preparation steps necessary, i.e. installing `testthat`, `devtools` and `roxygen2` packages.  See https://github.com/bigdatagenomics/adam/issues/1672#issuecomment-326047355 and thread for details.